website/src/views/Profile/__snapshots__/Profile.test.jsx.snap

```diff
@@ -1,6 +1,12822 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Profile View Expect the Profile page to match snap shot 1`] = `
+exports[`Profile View when userData is defined as student. Empty university with empty requested Class Expect the Profile page to match snap shot 1`] = `
```

Comment:
Since yuhao test multiple case for the profile page. Jest generated a huge snapshot file. So I just add a critical label to this PR.